packages/instant/src/redux/reducer.ts

```diff
@@ -121,3 +130,24 @@ export const reducer = (state: State = INITIAL_STATE, action: Action): State =>
             return state;
     }
 };
+
+const doesBuyQuoteMatchState = (buyQuote: BuyQuote, state: State): boolean => {
```

Comment:
maybe put this in a util file? don't feel strongly about it.

Reply:
will just leave it